cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go

```diff
@@ -35,42 +35,169 @@ const (
 	usbController          uint16 = 23
 )
 
-//Full list: https://vdc-download.vmware.com/vmwb-repository/dcr-public/da47f910-60ac-438b-8b9b-6122f4d14524/16b7274a-bf8b-4b4c-a05e-746f2aa93c8c/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html
+type osInfo struct {
+	description      string
+	importerOSIDs    []string
+	nonDeterministic bool
+}
```

Comment:
Can you add some comments for `description` and `nonDeterministic`? (For example, to help someone needing to add more OS mappings in a couple of years)

Reply:
Added descriptions